pennylane/qchem/molecule.py

```diff
@@ -130,6 +131,17 @@ def __init__(
 
         self.mo_coefficients = None
 
+    def __repr__(self):
+        """Returns the molecule representation in string format"""
+
+        elements, counter, flags = set(self.symbols), collections.Counter(self.symbols), []
+        if counter["C"]:
+            flags = ["C", "H"] if counter["H"] else ["C"]
+        ordered_elems = flags + list(sorted(elements.difference(set(flags))))
+        formula = "".join([x + str(counter[x]) if counter[x] > 1 else x for x in ordered_elems])
+
+        return f"<Molecule = {formula} charge: {self.charge} basis: {self.basis_name.upper()} n_orbs: {self.n_orbitals} n_elecs: {self.n_electrons}>"
```

Comment:
```suggestion
        return f"<Molecule = {formula}, charge: {self.charge}, basis: {self.basis_name.upper()}, n_orbs: {self.n_orbitals}, n_elecs: {self.n_electrons}>"
```
tests will need slight updating if you take this change 😰

Reply:
Agree with the commas. I also suggest using `orbitals` and `electrons` instead of `n_orbs` and `n_elecs`. Also, seems better to me to capitalise the first letter in all names similar to Molecule (or maybe decapitalize all):

`<Molecule = H3N, Charge: 1, Basis: STO-3G, Orbitals: 8, Electrons: 9>`